project/ISSUES.md

```diff
@@ -136,6 +136,17 @@ there are several other labels that are be useful and/or have special purpose.
   only requires limited understanding of the larger Galaxy framework and
   ecosystem.  This is useful for encouraging new contributors.
 
+* `minor` is a special tag used to generate release notes. It should
+  only be applied to pull requests made by committers that fix
+  functionality modified during the same release cycle. Such fixes are
```

Comment:
I see this as overly complex specification, I have serious trouble understanding this. Can we just say 'thi is noteworthy in release notes'?